src/server/api/user/updateUser.ts

```diff
@@ -6,7 +6,7 @@ import { Requests } from '@server/utils'
 export const updateUser = async (req: Request, res: Response) => {
   try {
     const profilePicture = req.file
-    const userToUpdate = req.body.user
+    const userToUpdate = JSON.parse(req.body.user)
```

Comment:
isnt it already parsed at this point

Reply:
no, it returned an error, that's why I fixed it this way